src/utils/user/getters.ts

```diff
@@ -76,6 +76,12 @@ export const getUserByEmail = async (email: string) => {
 };
 
 export const getUserByTicket = async (ticket: string) => {
+
+  // return null if ticket is empty
+  if (ticket === "") {
+    return null;
+  }
+
```

Comment:
and add `undefined` to the parameter type here and check if `ticket` is set:

```js
export const getUserByTicket = async (ticket: string | undefined) => {

if (!ticket) {
  return null;
}
```

Reply:
done!